react/features/conference/components/native/NavigationBar.js

```diff
@@ -55,26 +76,66 @@ class NavigationBar extends Component<Props> {
                 <SafeAreaView
                     pointerEvents = 'box-none'
                     style = { styles.navBarSafeView }>
-                    <View
-                        pointerEvents = 'box-none'
-                        style = { styles.navBarWrapper }>
-                        <PictureInPictureButton
-                            styles = { styles.navBarButton } />
-                        <View
-                            pointerEvents = 'box-none'
-                            style = { styles.roomNameWrapper }>
-                            <Text
-                                numberOfLines = { 1 }
-                                style = { styles.roomName }>
-                                { this.props._meetingName }
-                            </Text>
-                        </View>
-                    </View>
+                    { _navBarVisible && this._renderNavigationBar() }
+                    { _notificationsVisible
+                            && this._renderNotificationsContainer() }
                 </SafeAreaView>
             </View>
         );
     }
 
+    /**
+     * Renders a container for notifications to be displayed by the
+     * base/notifications feature.
+     *
+     * @private
+     * @returns {React$Element}
+     */
+    _renderNotificationsContainer() {
```

Comment:
I understand why you did it, but I really don't like these two features to be this tied. Can you please give it a shot and decouple them and render the notifications seperate from the navbar?

Reply:
Yes, I will try that, I don't like this neither. I have actually posted this PR just to get feedback about those 1 liner notifications in general, but didn't know about Github's draft feature at that time.